### PR TITLE
Bug #751 fix

### PR DESCRIPTION
**Pull request by Dzejkop**
### Fixed
- The error on OSX caused by downloading the patcher into location without necessary permissions.

### CODE REVIEW

File: patchkit-launcher-qt-src/src/locations.cpp [hunk 1]
```diff
@@ -127,7 +132,7 @@ QString Locations::patcherDirectoryPath() const
 
 QString Locations::patcherInstallationInfoFilePath() const
 {
-    return QDir::cleanPath(patcherDirectoryPath() + "/" + Config::patcherDirectoryName);
+    return QDir::cleanPath(patcherDirectoryPath() + "/" + Config::patcherInstallationInfoFileName);
```
Comment (witcher112) [inline patchkit-launcher-qt-src/src/locations.cpp]: I've researched the code and found some places that highly depend on file located under that path. Please make a modification that would support previous file location.

Comment (Dzejkop) [reply patchkit-launcher-qt-src/src/locations.cpp]: I disagree, while backwards compatibility with previous Patcher versions is required, backwards compatibility with previous Launcher versions is not, as users won't typically update their launchers.

Comment (witcher112) [reply patchkit-launcher-qt-src/src/locations.cpp]: Ok, I agree with that. @genail please share your opinion about it, I'd like to know what do you think about it.

Comment (genail) [reply patchkit-launcher-qt-src/src/locations.cpp]: I also agree.

**Comment by witcher112** [comment]
@Dzejkop if you decide that pull request is ready, please merge it. From our side everything's okay.

**Pull request merged**